new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Installer.java

```diff
@@ -122,4 +137,45 @@ private String buildDownloadCommand() {
         params.fileName());
     return command;
   }
+
+  // Figure out the actual release number of the latest
+  static String getActualVersionIfNeeded(
+      String location,
+      String type,
+      String version
+  ) {
+    if (version == null || version.equalsIgnoreCase("latest")) {
+      String command = String.format(
+          "curl -fL %s/releases/latest -o %s/%s-%s", 
+          location,
+          DOWNLOAD_DIR,
+          type,
+          TMP_FILE_NAME);
+      
+      if (!Command.withParams(defaultCommandParams().command(command)).execute()) {
+        return null;
+      }
+
+      command = String.format(
+          "cat %s/%s-%s | grep 'releases/download' | awk '{ split($0,a,/href=\"/);%s | %s", 
+          DOWNLOAD_DIR, 
+          type, 
+          TMP_FILE_NAME, 
+          " print a[2] }'", 
+          " cut -d/ -f 6"); 
+    
+      CommandParams params = 
+          defaultCommandParams()
+          .command(command)
+          .saveStdOut(true)
+          .redirect(true);
+
+      if (Command.withParams(params).execute()) {
+        return params.stdOut();
+      } else {
+        return null;
```

Comment:
Same comment

Reply:
fixed with a severe log message and throwing a RuntimeException. The caller catches the exception and returns false to fail the test case.